provisioning/JJG-Ansible-Windows/windows.sh

```diff
@@ -17,7 +34,7 @@ YUM=$(which yum 2>/dev/null)
 APT_GET=$(which apt-get 2>/dev/null)
 
 # Make sure Ansible playbook exists.
-if [ ! -f "/vagrant/$ANSIBLE_PLAYBOOK" ]; then
```

Comment:
If this is acceptable, we need to commit this to https://github.com/geerlingguy/JJG-Ansible-Windows first. I think forcing  the path to be within `/vagrant` to be an unnecessary restriction. Also it looks weird when you call the script with an absolute-looking path that's actually relative.